### ISSUE TITLE
[FR] Clear “Handled” stats only

### ISSUE THREAD

**Issue by furioursus** [nystudio107/craft-retour#17]
Just a thought, but when the critical mass of results/items hits 1000 or whatever the maximum number has been set to, it might be helpful to clear only the stats that have been marked as handled. This way, if you are in the middle of a huge migration as I am, you can pay attention to unresolved items that might have fewer hits and be less likely to show up again in the immediate future.

**Comment by khalwat** [comment]
So these stats will always be sorted in by `hitCount` -- when the stats are trimmed, it trims the oldest stats with the least `hitCount` -- so they are almost never things you care about.

**Comment by furioursus** [comment]
Absolutely. 95% of the time I’m in agreement with that. It’s obviously not a huge issue since I can just download the report before emptying it out.

**Comment by furioursus** [comment]
(Fat thumbed, whoops)
